Detectors/PHOS/base/include/PHOSBase/Hit.h

```diff
@@ -75,9 +75,9 @@ class Hit : public o2::BasicXYZEHit<float>
 
   /// \brief Get the initial energy of the primary particle entering PHOS
   /// \return Energy of the primary particle entering PHOS
-  Double_t GetInitialEnergy() const { return mInitialEnergy; }
+  Double_t getInitialEnergy() const { return mInitialEnergy; }
```

Comment:
again, do you need double for energy?

Reply:
Dear Ruben, 
I hope I implemented all your suggestions in the current version. 
     Dmitri